src/game/boe.actions.cpp

```diff
@@ -2627,21 +2627,24 @@ void switch_pc(short which) {
 	}
 }
 
-void drop_pc(short which) {
-	std::string choice;
-	
-	choice = cChoiceDlog("delete-pc-confirm",{"yes","no"}).show();
-	if(choice == "no") {
-		add_string_to_buf("Delete PC: Cancelled.");
-		return;
+void drop_pc() {
```

Comment:
I kinda think it makes more sense to keep the function taking one parameter… or maybe rename it?

Reply:
If it's kept keeping only one parameter, then the choice of PC from 1-6 will have to be done outside of `drop_pc()` at every callsite, which is icky. We're only calling `drop_pc()` from one place, but choosing which PC to drop is an inherent part of the `drop_pc()` process, so why not put it in the function?

Reply:
Well, to me the name "drop_pc" implies it should take a parameter…

Reply:
maybe `handle_drop_pc()`?

Reply:
Sure.